config/default.json

```diff
@@ -1,26 +1,16 @@
 {
   "services": {
     "ceramic": {
-      "mode": "none",
-      "apiURL": "http://thisisaurl:1234",
-      "anchorServiceAPI": "http://thisisaurl:1234",
-      "ethereumRpc": "http://thisisaurl:1234",
-      "network": "dev-unstable"
+      "mode": "none"
```

Comment:
hmm... I thought this wouldn't work because it would break the way the typescript code for the `config` object is auto-generated, since now it won't know about the other config options?

Reply:
@stbrody So should I just be setting them to null then?

Reply:
I think splitting up configs will avoid this so I’m gonna move to that